build.sbt

```diff
@@ -11,15 +11,73 @@ ThisBuild / scmInfo := Some(ScmInfo(
 
 ThisBuild / publishAsOSSProject := true
 
+lazy val quasarVersion =
+  Def.setting[String](managedVersions.value("precog-quasar"))
+
+lazy val quasarPluginJdbcVersion =
+  Def.setting[String](managedVersions.value("precog-quasar-plugin-jdbc"))
+
+val Specs2Version = "4.9.4"
+
 // Include to also publish a project's tests
 lazy val publishTestsSettings = Seq(
   Test / packageBin / publishArtifact := true)
 
 lazy val root = project
   .in(file("."))
   .settings(noPublishSettings)
-  .aggregate(core)
+  .aggregate(core, datasource)
 
 lazy val core = project
   .in(file("core"))
-  .settings(name := "quasar-plugin-avalanche")
+  .settings(
+    name := "quasar-plugin-avalanche",
+
+    libraryDependencies ++= Seq(
+      "com.precog" %% "quasar-plugin-jdbc" % quasarPluginJdbcVersion.value,
+      "org.specs2" %% "specs2-core" % Specs2Version % Test
+    ),
+
+    // FIXME: how do we include this without assembly?
+    // Assemble a "fat" jar consisting of the plugin and the unmanaged iijdbc.jar
```

Comment:
Can you elaborate more on the problem here? Is it that we cannot have a dependency managed by sbt because there is no release of this hosted in an artifact repository?

Reply:
Yes, exactly, the driver `jar` doesn't appear to be published anywhere so we have to include it manually.

I'm not sure if this is _that_ bad a use of `sbt-assembly`, but @djspiewak had just mentioned desiring to avoid it the other day, so I added the `FIXME` as a reminder to investigate at some point.

Reply:
Yeah I think in this case, it's an okay and probably inevitable use of sbt-assembly.